server/src/models/Burger.js

```diff
@@ -27,6 +28,26 @@ class Burger extends Model {
           from: "burgers.restaurantId",
           to: "restaurants.id"
         }
+      },
+      reviews: {
+        relation: Model.HasManyRelation,
+        modelClass: Review,
+        join: {
+          from: "burgers.id",
+          to: "reviews.burgerId"
+        }
+      },
+      users: {
```

Comment:
I don't think we need to define this association because it's most likely not going to be used